### PR TITLE
Pagination component

### PR DESCRIPTION
**Pull request by KlonD90**

### CODE REVIEW

File: src/components/pagination/Pagination.tsx [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+      <ul
+        className={css({
+          display: "flex",
+          flexDirection: "row",
+          listStyle: "none",
+          gap: "8px",
+        })}
+      >
+        <PageElement
+          key="prev"
```
Comment (ukorvl) [inline src/components/pagination/Pagination.tsx:121]: Why do we need key?

Comment (KlonD90) [reply src/components/pagination/Pagination.tsx:144]: just help react

File: src/components/pagination/Pagination.tsx [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+    for (const page of pages) {
+      if (page - prevPage > 1) {
+        elements.push(
+          <Button kind={BUTTON_KIND.tertiary} size={buttonSize} disabled>
+            ...
+          </Button>
+        );
+      }
+      elements.push(
+        <li key={page} className="page-item">
```
Comment (ukorvl) [inline src/components/pagination/Pagination.tsx:111]: Here `<li>` elements are duplicated, one inside another

Comment (ukorvl) [inline src/components/pagination/Pagination.tsx:111]: What is `className="page-item"` used for?

Comment (KlonD90) [reply src/components/pagination/Pagination.tsx]: fix it

Comment (KlonD90) [reply src/components/pagination/Pagination.tsx]: remove it

File: src/components/button/types.ts [hunk 3]
```diff
@@ -24,6 +24,7 @@ export type ButtonCommonProps = Omit<BaseButtonProps, "kind" | "shape" | "size">
   isLoading?: boolean;
   className?: string;
   role?: AriaRole;
+  tabIndex?: number;
```
Comment (ukorvl) [inline src/components/button/types.ts:27]: I think this is not the best way to add props one by one here. Better use `Component props<"button">`

File: src/components/pagination/Pagination.tsx [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+    };
+  }
+  if (linkMapper && !disabled) {
+    return (
+      <li>
+        <a
+          href={linkMapper(page)}
+          onClick={(e) => {
+            e.preventDefault();
+            if (pageHandler) pageHandler(page);
```
Comment (ukorvl) [inline src/components/pagination/Pagination.tsx:26]: Why do we need additional pageHandler here, though we already have one in `buttonProps`?

File: src/components/button/types.ts (hunk 3 above)
Comment (KlonD90) [reply src/components/button/types.ts]: updated it

**Review by ukorvl — APPROVED** [review]

**Pull request merged**